vizro-core/tests/unit/vizro/models/_components/test_figure.py

```diff
@@ -0,0 +1,96 @@
+"""Unit tests for vizro.models.Figure."""
+
+import dash_bootstrap_components as dbc
+import pytest
+from asserts import assert_component_equal
+from dash import dcc, html
+
+try:
+    from pydantic.v1 import ValidationError
+except ImportError:  # pragma: no cov
+    from pydantic import ValidationError
+
+import vizro.models as vm
+from vizro.figures import kpi_card
+from vizro.managers import data_manager
+
+
+@pytest.fixture
+def kpi_card_with_dataframe(gapminder):
+    return kpi_card(
+        data_frame=gapminder,
+        value_column="lifeExp",
+        agg_func="mean",
+        title="Life Expectancy",
+        value_format="{value:.3f}",
+    )
+
+
+@pytest.fixture
+def kpi_card_with_str_dataframe():
+    return kpi_card(
+        data_frame="gapminder",
+        value_column="lifeExp",
+        agg_func="mean",
+        title="Life Expectancy",
+        value_format="{value:.3f}",
+    )
+
+
+class TestFigureInstantiation:
```

Comment:
Should we add the `test_getitem_known_args` and the `test_getitem_unknown_args` (something similar we have in `test_graph.py`) under the `class TestDunderMethodsFigure`?